protobuf-codegen/src/gen/mod_rs.rs

```diff
@@ -4,8 +4,8 @@ use crate::gen::code_writer::CodeWriter;
 pub(crate) fn gen_mod_rs(mods: &[String]) -> compiler_plugin::GenResult {
     let v = CodeWriter::with_no_error(|w| {
         w.comment(&format!("{}generated", "@"));
-        w.write_line("");
         for m in mods {
+            w.write_line("");
```

Comment:
Now it writes a line between each mod? Doesn't look right.

Reply:
The formatter won't sort mods when they are empty line delimited

Reply:
Wouldn't it be better to just sort mods?

Reply:
Sure, if u prefer